pkg/csi/csitest/driver/driver.go

```diff
@@ -0,0 +1,91 @@
+package driver
+
+import (
+	"net"
+	"sync"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+)
+
+// CSIDriverServers is a unified driver component with both Controller and Node
```

Comment:
please add link here where we copied / or inspired so in future we can find it easy.